OpenRA.Mods.Common/Lint/CheckTranslationReference.cs

```diff
@@ -292,6 +293,64 @@ static TranslationKeys GetUsedTranslationKeysInMap(Map map, Action<string> emitW
 				}
 			}
 
+			var hotkeyTranslationFields =
+				Utility.GetFields(typeof(HotkeyDefinition))
+					.Select(f => (Field: f, TranslationReference: Utility.GetCustomAttributes<TranslationReferenceAttribute>(f, true).SingleOrDefault()))
+					.Where(x => x.TranslationReference != null)
+					.ToArray();
+			testedFields.AddRange(hotkeyTranslationFields.Select(x => x.Field));
+			foreach (var hotkeyDefinition in modData.Hotkeys.Definitions)
+			{
+				foreach (var (field, translationReference) in hotkeyTranslationFields)
+				{
+					var keys = LintExts.GetFieldValues(hotkeyDefinition, field, translationReference.DictionaryReference);
+					foreach (var key in keys)
+						usedKeys.Add(key, translationReference, $"`{field.ReflectedType.Name}.{field.Name}`");
+				}
+			}
+
+			var keycodeTranslationFields =
+				new[]
+				{
+					typeof(KeycodeExts).GetField("KeycodeTranslationKeys", BindingFlags.NonPublic | BindingFlags.Static),
+					typeof(ModifiersExts).GetField("Cmd", BindingFlags.NonPublic | BindingFlags.Static),
+					typeof(ModifiersExts).GetField("ModifierTranslationKeys", BindingFlags.NonPublic | BindingFlags.Static)
```

Comment:
this magic is really dirty, how could we reference these directly?

Reply:
The whole lint pass is based on relecting over fields with the `TranslationReferenceAttribute` - not sure what makes this part any different?

Reply:
do nameof
```csharp
KeycodeExts.KeycodeTranslationKeys;
ModifiersExts.ModifierTranslationKeys;
ModifiersExts.Cmd;
```

instead. Better to make them public than bind them like this

Reply:
or maybe another way of including these in the lint

Reply:
Made them public.